annotation_converters/objectron_2_coco.py

```diff
@@ -0,0 +1,198 @@
+import argparse
+import json
+import os
+from os import path as osp
+
+from tqdm import tqdm
+import cv2 as cv
+import numpy as np
+
+from objectron_helpers import load_annotation_sequence, grab_frames, get_video_frames_number
+
+lists_root_path = osp.abspath(os.path.join(osp.dirname(__file__), '../3rdparty/Objectron/index'))
+
+
+def load_video_info(data_root, subset, classes):
+    videos_info = []
+    avg_vid_len = 0
+    for cl in classes:
+        with open(osp.join(lists_root_path, f'{cl}_annotations_{subset}'), 'r') as f:
+            for line in f:
+                ann_path = osp.join(data_root, 'annotation' + osp.sep + line.strip() + '.pbdata')
+                ann = load_annotation_sequence(ann_path) # object class can be incorrect in annoatation
+                for item in ann:
+                    item[1] = cl
```

Comment:
Have you ensured, that, in the case when class incorrect in annotation, everything else except the class name is correct?

Reply:
Only for several videos. I'll add a warning message here to print names of suspectful videos. Later they could be manually checked.